pkg/template/apis/template/types.go

```diff
@@ -108,10 +108,24 @@ type TemplateInstanceSpec struct {
 // TemplateInstanceRequester holds the identity of an agent requesting a
 // template instantiation.
 type TemplateInstanceRequester struct {
```

Comment:
why can't we use the pkg/authentication/user/DefaultInfo struct here instead of creating our own?